src/theme/zIndex.js

```diff
@@ -0,0 +1 @@
+export const overlay = 50
```

Comment:
let's do a `export default {}` here like `colors` and `fontFamilies`. If we ever need to make this configurable it will be more painful with named exports